source/features/prevent-pr-commit-link-loss.tsx

```diff
@@ -46,7 +46,19 @@ void features.add({
 	screenshot: 'https://user-images.githubusercontent.com/1402241/82131169-93fd5180-97d2-11ea-9695-97051c55091f.gif'
 }, {
 	include: [
-		pageDetect.hasComments
+		pageDetect.hasRichTextEditor
 	],
 	init
+}, {
+	include: [
+		pageDetect.isNewIssue,
+		pageDetect.isCompare
+	],
+	exclude: [
+		() => select.exists('.form-actions')
+	],
+	init: () => {
+		// New issues and Pull Requests are missing the form-actions selector, this avoids having a separate logic just for them.
+		select('#new_issue tab-container, #new_pull_request tab-container')?.after(<div className="form-actions"/>);
```

Comment:
@yakov116 If you reset to your first commit only, and then apply this suggestion, it should work :)

The culprit is `float: left;` of the "Allow edits" checkbox, so we need a `clear: left;` before our warning, which is added by the `clearfix` class.
The utility classes `pb-0` and `pt-2` fix the spacing around the warning.

```suggestion
		const formTabContainer = select('#new_issue tab-container, #new_pull_request tab-container');
		formTabContainer?.classList.add('clearfix');
		formTabContainer?.after(<div className="form-actions pb-0 pt-2"/>);
```

Reply:
Thanks!! will try it in a few